src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeregningBarnetilsynService.kt

```diff
@@ -22,6 +29,37 @@ class BeregningBarnetilsynService {
                 }
                 .mergeSammenhengendePerioder()
     }
+
+    fun List<PeriodeMedBeløpDto>.tilPerioder(): List<Periode> =
+            this.map {
+                it.tilPeriode()
+            }
+
+    private fun validerGyldigePerioder(utgiftsperioder: List<Periode>,
+                                       kontantstøttePerioder: List<Periode>,
+                                       tilleggsstønadsperioder: List<Periode>) {
+        brukerfeilHvis(utgiftsperioder.isEmpty()) { "Ingen utgiftsperioder" }
+        brukerfeilHvis(harUrelevantReduksjonsPeriode(utgiftsperioder, kontantstøttePerioder)){ "Urelevant kontantstøtteperiode kan fjernes" }
+        brukerfeilHvis(harUrelevantReduksjonsPeriode(utgiftsperioder, tilleggsstønadsperioder)){ "Urelevant tilleggsstønadsperiode kan fjernes" }
+        brukerfeilHvis(utgiftsperioder.harOverlappende()) { "Utgiftsperioder $utgiftsperioder overlapper" }
+        brukerfeilHvis((kontantstøttePerioder.harOverlappende())) { "Kontantstøtteperioder $kontantstøttePerioder overlapper" }
+        brukerfeilHvis((tilleggsstønadsperioder.harOverlappende())) { "Tilleggsstønadsperioder $tilleggsstønadsperioder overlapper" }
```

Comment:
Noen litt lange rader, og hadde vært fint med litt space? 
```suggestion
        brukerfeilHvis(utgiftsperioder.isEmpty()) { "Ingen utgiftsperioder" }
        
        brukerfeilHvis(harUrelevantReduksjonsPeriode(utgiftsperioder, kontantstøttePerioder)){ "Urelevant kontantstøtteperiode kan fjernes" }
        brukerfeilHvis(harUrelevantReduksjonsPeriode(utgiftsperioder, tilleggsstønadsperioder)){ "Urelevant tilleggsstønadsperiode kan fjernes" }
        
        brukerfeilHvis(utgiftsperioder.harOverlappende()) { "Utgiftsperioder $utgiftsperioder overlapper" }
        brukerfeilHvis((kontantstøttePerioder.harOverlappende())) { "Kontantstøtteperioder $kontantstøttePerioder overlapper" }
        brukerfeilHvis((tilleggsstønadsperioder.harOverlappende())) { "Tilleggsstønadsperioder $tilleggsstønadsperioder overlapper" }
```